mlflow/tracking/fluent.py

```diff
@@ -2035,7 +2038,7 @@ def search_logged_models(
         )
 
 
-def log_outputs(models: Optional[List[LoggedModelOutput]] = None):
+def log_outputs(models: List[LoggedModelOutput]):
```

Comment:
```suggestion
def log_outputs(*, models: List[LoggedModelOutput]):
```

Reply:
so we want to make required arguments keyword-only as well?

Reply:
There is a pro and a con in this case.

Con: It might be a bit inconvenient because you always need to type `models`.
Pro: Easier to tell what the passed argument is.

Reply:
Yeah, we should make required args keyword-only, in case we want to make them optional in the future. This is very common.